CODE_OF_CONDUCT.md

```diff
@@ -49,6 +49,7 @@ These actions are explicitly forbidden in OpenStreetMap Carto spaces:
 - Insulting, demeaning, hateful, or threatening remarks.
 - Discrimination based on age, disability, gender, nationality, ethnicity, religion, sexuality, or similar personal characteristic.
 - Bullying or systematic harassment.
+- Posting (or threatening to post) other people’s personally identifying information ("doxxing")
```

Comment:
The case that led to this PR involved exposing someone's location/address. Would it be useful to include that as a clarifying example of personally identifying information (PII) as meant here? The definition of PII seems to vary quite a bit depending on the source from 'information that identifies as specific person' to 'identifiable information attributed to a specific person'. A home location/address is the latter.

Reply:
> The case that led to this PR involved exposing someone's location/address

though accuracy was limited to town and type of location remained unknown

Reply:
The way you word it now means it comes across as not being applicable to the incident it is a response too. That's not necessarily a problem, but seems weird.

'Doxxing' is a neologism (unlike the other terms you list) and PII has definitions that very in scope depending on the legal framework that refers to it (like the GDPR).

Reply:
As defined here, someone posting my name and location with my explicit permission would be doxxing me, and thus prohibited. Additionally, some users are publicly disclose their location in their profile[^1][^2]. These users are publicising certain information about themselves, and, when relevant to an issue, I see no reason not to mention the information they publish.

When defining dox, Oxford and Wikipedia both include some notion of the information being private and there normally being some malice. The definition here has neither.

[^1]: https://www.openstreetmap.org/user/Mateusz%20Konieczny has real name and city
[^2]: https://www.openstreetmap.org/user/pnorman has only city (which is incorrect), but real name is in code commits